docs/en/operations/system_tables.md

```diff
@@ -742,17 +742,22 @@ WHERE
 
 If this query doesn't return anything, it means that everything is fine.
 
-## system.settings
+## system.settings {#system-tables-system-settings}
 
-Contains information about settings that are currently in use.
-I.e. used for executing the query you are using to read from the system.settings table.
+Contains information about settings that are currently in use. 
```

Comment:
1. All the settings excepting server configuration parameters are always in use.
2. Settings introduction - http://130.193.44.42/elenbaskakova/clickhouse-fork/docs/build/docs/en/operations/settings/